Build Attach with Xcode 11.7

Related to #137, it seems that the symbol `___isPlatformVersionAtLeast` has been removed in Xcode 11.7. 

StatusBar is currently built with Xcode 11.3 adds it, and causes this error: `Undefined symbols for architecture arm64: "___isPlatformVersionAtLeast"`.

--- a/.travis.yml
+++ b/.travis.yml
@@ -6,7 +6,7 @@ sudo: false
 
 # use Mac OS X as operating system to build Android and iOS native libraries
 os: osx
-osx_image: xcode11.3
+osx_image: xcode11.7
 
 before_script:
   - bash .ci/android.sh

--- a/.travis.yml
+++ b/.travis.yml
@@ -12,7 +12,7 @@ before_script:
   - bash .ci/android.sh
   - export ANDROID_SDK=$HOME/android-sdk
   - export ANDROID_NDK=$HOME/android-sdk/ndk-bundle
-  - export JAVA_HOME=/Library/Java/JavaVirtualMachines/openjdk-14.jdk/Contents/Home
+  - export JAVA_HOME=/Library/Java/JavaVirtualMachines/openjdk-14.0.2.jdk/Contents/Home
 
 # run maven build
 script:

